django/db/models/functions/math.py

```diff
@@ -0,0 +1,121 @@
+from django.db.models import FloatField, Func, Transform
+
+
+class Abs(Transform):
+    function = 'ABS'
+    lookup_name = 'abs'
+
+
+class ACos(Transform):
+    function = 'ACOS'
+    lookup_name = 'acos'
+    output_field = FloatField()
+
+
+class ASin(Transform):
+    function = 'ASIN'
+    lookup_name = 'asin'
+    output_field = FloatField()
+
+
+class ATan(Transform):
+    function = 'ATAN'
+    lookup_name = 'atan'
+    output_field = FloatField()
+
+
+class ATan2(Func):
+    function = 'ATAN2'
+    arity = 2
+    output_field = FloatField()
+
+
+class Ceil(Transform):
+    function = 'CEILING'
+    lookup_name = 'ceil'
+
+
+class Cos(Transform):
+    function = 'COS'
+    lookup_name = 'cos'
+    output_field = FloatField()
+
+
+class Cot(Transform):
+    function = 'COT'
+    lookup_name = 'cot'
+    output_field = FloatField()
+
+
+class Degrees(Transform):
+    function = 'DEGREES'
+    lookup_name = 'degrees'
+    output_field = FloatField()
+
+
+class Exp(Transform):
+    function = 'EXP'
+    lookup_name = 'exp'
+
+
+class Floor(Transform):
+    function = 'FLOOR'
+    lookup_name = 'floor'
+
+
+class Ln(Transform):
+    function = 'LN'
+    lookup_name = 'ln'
+
+
+class Log(Func):
+    function = 'LOG'
+    arity = 2
+    output_field = FloatField()
+
+
+class Mod(Func):
+    function = 'MOD'
+    arity = 2
+    output_field = FloatField()
+
+
+class Pi(Func):
+    function = 'PI'
```

Comment:
`PI` doesn't exist on Oracle. It is a constant, therefore we can use `math.pi`.

Reply:
@felixxm, please check this link
https://docs.oracle.com/javadb/10.8.3.0/ref/rreffuncpi.html